cmd/celestia/version.go

```diff
@@ -1,15 +1,20 @@
+// +build: !test
 package main
 
 import (
 	"fmt"
 	"runtime"
 
+	_ "embed"
+
 	"github.com/spf13/cobra"
 )
 
 var (
-	buildTime       string
-	lastCommit      string
+	buildTime string
+	//go:embed lastCommit.txt
```

Comment:
we need to document this lastCommit.txt here. 
And exclude it from the the go test executor

Reply:
see my commetn

Reply:
Hmm the no build directive for tests seems to not work @jrmanes do you mind looking into it?

Reply:
I can take a look 👌